tsconfig.json

```diff
@@ -1,11 +1,11 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2021",
```

Comment:
Needed to use `string.replaceAll` here https://github.com/appit-online/youtube-search/pull/19/files#diff-82164b7feebd2bb287f266221d19e985054c7ccf2f3b0e45a6421a5b0ef9e823L23